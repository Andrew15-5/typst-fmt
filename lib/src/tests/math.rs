use super::*;

make_test!(mathblock1,
r#"$
    #xx(a,b) &= 1 \
    &= 2 \

$"#);


make_test!(mathblock2,
r#"$x$"#);


make_test!(mathblock3,
r#"$
    #xx(a,b) &= 1 \
    &= 2 \
    &=3 \
    &=4 \

$"#);

make_test!(mathblock4,
r#"$
    #xx(a,b) &= 1 &=3 \
    &= 2 &=2 \
    &=3 \
    &=4 \
    &=5 \
    $"#);

make_test!(mathblock5,
r#"$
#xx(a,b)
    &= 1 &=3 \
    &= 2 &=2 \
    &=3 \
    &=4 \
    &=5 \
    $"#);
