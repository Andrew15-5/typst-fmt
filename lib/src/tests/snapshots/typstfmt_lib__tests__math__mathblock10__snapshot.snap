---
source: lib/src/tests/math.rs
description: "INPUT\n===\n\"\\n$\\n  a&=b & c & d & e\\\\\\n  &= \\\"a really long string\\\" & pi \\\\\\n  &= a & #xx & d & e \\\\\\n  &= \\\"an even longer string!!\\\" & y\\n$\"\n===\n\n$\n  a&=b & c & d & e\\\n  &= \"a really long string\" & pi \\\n  &= a & #xx & d & e \\\n  &= \"an even longer string!!\" & y\n$\n===\nFORMATTED\n===\n\n$\n  a&=b                          & c   & d & e\\\n   &= \"a really long string\"    & pi \\\n   &= a                         & #xx & d & e \\\n   &= \"an even longer string!!\" & y\n$"
expression: formatted
---
"\n$\n  a&=b                          & c   & d & e\\\n   &= \"a really long string\"    & pi \\\n   &= a                         & #xx & d & e \\\n   &= \"an even longer string!!\" & y\n$"
